rho/scancommand.py

```diff
@@ -50,57 +49,46 @@ def _create_ping_inventory(vault, vault_pass, profile_ranges, profile_port,
     success_map = defaultdict(list)
     best_map = defaultdict(list)
     mapped_hosts = set()
+    hosts_dict = dict()
```

Comment:
Out of curiosity I prefer the literal dict, instead of the constructor, because it is faster:

```python
In [1]: %timeit d = dict()
The slowest run took 102.70 times longer than the fastest. This could mean that an intermediate result is being cached.
10000000 loops, best of 3: 97.5 ns per loop

In [2]: %timeit d = {}
The slowest run took 95.34 times longer than the fastest. This could mean that an intermediate result is being cached.
10000000 loops, best of 3: 22.5 ns per loop
```